### ISSUE TITLE
ThemeEditor: What-If enhancements

### ISSUE THREAD

**Issue by djwisdom** [SerenityOS/serenity#13847]
### Current ThemeEditor:
![ThemeEditorWindows](https://user-images.githubusercontent.com/391659/166105407-944bbccd-0d7d-426d-8d54-1276ec4d8f7e.png)

### What-if with facility to see Tray, SerenityMenu and Tooltip
![ThemeEditorTraySerenityMenu](https://user-images.githubusercontent.com/391659/166105433-aafeaae7-e17b-4d14-9a15-c31a86cf6a83.png)

### What-if with facility to see Ruler/Gutter
![ThemeEditorHackStudio01](https://user-images.githubusercontent.com/391659/166105473-85545f0f-1adf-4c5d-8cc2-c6b6d92de477.png)

### What-if with facility to see SyntaxHighlighting per Theme
![ThemeEditorHackStudio02](https://user-images.githubusercontent.com/391659/166105489-c86a9e32-a107-4d48-aeba-792ebbd1d4ce.png)

### What-if with facility to see RubberBand Fill
![ThemeEditorRubberBandFill](https://user-images.githubusercontent.com/391659/166105513-9569861a-a447-4d63-a7f0-6a60eafdc283.png)

### Then this will be one awesome ThemeEditor indeed.  :^)

**Comment by Diogenes31** [comment]
I agree that these would be good things to be able to see, namely the tab highlight color, which I find to be lacking in the current theme-editor. :yakplus:

**Comment by djwisdom** [comment]
A move in the right direction, https://github.com/SerenityOS/serenity/pull/13937 is quite an exciting development :^)
